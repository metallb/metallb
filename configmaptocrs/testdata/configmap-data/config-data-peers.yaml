peers:
- my-asn: 64512
  peer-asn: 64512
  peer-address: 10.96.0.100
- my-asn: 64512
  peer-asn: 64512
  peer-address: 10.96.0.101
- peer-address: 10.0.0.1
  peer-asn: 64512
  my-asn: 64512
  peer-port: 179
  source-address: 10.0.0.2
  hold-time: 120s
  keepalive-time: 30s
  router-id: 1.2.3.4
  password: "yourPassword"
  bfd-profile: "bfdprofile1"
  ebgp-multihop: false
  node-selectors:
  - match-labels:
      kubernetes.io/hostname: prod-01
    match-expressions:
    - key: kubernetes.io/arch
      operator: In
      values: [amd64, arm]
bfd-profiles:
- name: bfdprofile1
  receive-interval: 280
  transmit-interval: 270
