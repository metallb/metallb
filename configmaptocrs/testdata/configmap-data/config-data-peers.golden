# This was autogenerated by MetalLB's custom resource generator.
apiVersion: metallb.io/v1beta2
kind: BGPPeer
metadata:
  creationTimestamp: null
  name: peer1
  namespace: metallb-system
spec:
  holdTime: 1m30s
  keepaliveTime: 0s
  myASN: 64512
  passwordSecret: {}
  peerASN: 64512
  peerAddress: 10.96.0.100
status: {}
---
apiVersion: metallb.io/v1beta2
kind: BGPPeer
metadata:
  creationTimestamp: null
  name: peer2
  namespace: metallb-system
spec:
  holdTime: 1m30s
  keepaliveTime: 0s
  myASN: 64512
  passwordSecret: {}
  peerASN: 64512
  peerAddress: 10.96.0.101
status: {}
---
apiVersion: metallb.io/v1beta2
kind: BGPPeer
metadata:
  creationTimestamp: null
  name: peer3
  namespace: metallb-system
spec:
  bfdProfile: bfdprofile1
  holdTime: 2m0s
  keepaliveTime: 30s
  myASN: 64512
  nodeSelectors:
  - matchExpressions:
    - key: kubernetes.io/arch
      operator: In
      values:
      - amd64
      - arm
    matchLabels:
      kubernetes.io/hostname: prod-01
  password: yourPassword
  passwordSecret: {}
  peerASN: 64512
  peerAddress: 10.0.0.1
  peerPort: 179
  routerID: 1.2.3.4
  sourceAddress: 10.0.0.2
status: {}
---
apiVersion: metallb.io/v1beta1
kind: BFDProfile
metadata:
  creationTimestamp: null
  name: bfdprofile1
  namespace: metallb-system
spec:
  echoMode: false
  passiveMode: false
  receiveInterval: 280
  transmitInterval: 270
status: {}
---
